test/e2e/run_test.go

```diff
@@ -1376,7 +1376,27 @@ USER mail`
 	})
 
 	It("podman run --tz", func() {
-		session := podmanTest.Podman([]string{"run", "--tz", "foo", "--rm", ALPINE, "date"})
+		testDir := filepath.Join(podmanTest.RunRoot, "tz-test")
+		err := os.MkdirAll(testDir, 0755)
+		Expect(err).To(BeNil())
+
+		tzFile := filepath.Join(testdir, "tzfile.txt")
+		file, err := os.Create(tzFile)
+		Expect(err).To(BeNil())
+		defer file.Close()
+
+		_, err = file.WriteString("Hello")
+		Expect(err).To(BeNil())
```

Comment:
Don't you need a `file.Close()` here? Without it, you can't be guaranteed that the `Hello` will be flushed when `podman run` tries to read it.

Reply:
I've dropped the defer, thx.

Reply:
You have to write before close.

Reply:
Yep, copied it below, removed the defer above and should have killed it entirely.